go.sum

```diff
@@ -287,10 +287,8 @@ github.com/pmezard/go-difflib v1.0.0 h1:4DBwDE0NGyQoBHbLQYPwSUPoCMWR5BEzIk/f1lZb
 github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
 github.com/posener/complete v1.1.1/go.mod h1:em0nMJCgc9GFtwrmVmEMR/ZL6WyhyjMBndrE9hABlRI=
 github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4/go.mod h1:xMI15A0UPsDsEKsMN9yxemIoYk6Tm2C1GtYGdfGttqA=
-github.com/rockset/rockset-go-client v0.12.3 h1:Iq9bwJiA0W97CNi9Po9Xky/0culPiVctwamjzyBGv1A=
-github.com/rockset/rockset-go-client v0.12.3/go.mod h1:Apc/bRmbRrpBvgwtXFH3L3Afyk1tDSBZATtv0Nmb2LI=
-github.com/rockset/rockset-go-client v0.12.4 h1:0YkjFk/EN9/Rx2Lz3UNg4q35Pv+BM6UEJMtBf34/ko4=
-github.com/rockset/rockset-go-client v0.12.4/go.mod h1:Apc/bRmbRrpBvgwtXFH3L3Afyk1tDSBZATtv0Nmb2LI=
```

Comment:
Why did it depend on multiple version before?

Reply:
lack of cleanup - `go mod tidy` fixed it